README.md

```diff
@@ -15,128 +15,73 @@ for an independent review.
-  between the current branch and its parent
-- [git town repo](/documentation/commands/repo.md) - view the repository
-  homepage
+- [git town ship](https://www.git-town.com//documentation/commands/ship.md) -
+  delivers a completed feature branch and removes it
+
+#### Advanced development commands
+
+- [git town kill](https://www.git-town.com//documentation/commands/kill.md) -
+  removes a feature branch
+- [git town prune-branches](https://www.git-town.com//documentation/commands/prune-branches.md) -
+  delete all merged branches
+- [git town rename-branch](https://www.git-town.com//documentation/commands/rename-branch.md) -
+  rename a branch
+- [git town repo](https://www.git-town.com//documentation/commands/repo.md) -
+  view the repository homepage
+
+#### Nested feature branches
+
+- [git town append](https://www.git-town.com//documentation/commands/append.md) -
+  insert a new branch as a child of the current branch
+- [git town prepend](https://www.git-town.com//documentation/commands/prepend.md) -
+  insert a new branch between the current branch and its parent
+- [git town set-parent-branch](https://www.git-town.com//documentation/commands/set-parent-branch.md) -
+  updates a branch's parent
 
-**Git Town Configuration**
+#### Git Town Configuration
 
-- [git town config](/documentation/commands/config.md) - displays or updates
-  your Git Town configuration
-- [git town new-branch-push-flag](/documentation/commands/new-branch-push-flag.md) -
+- [git town config](https://www.git-town.com//documentation/commands/config.md) -
+  displays or updates your Git Town configuration
+- [git town new-branch-push-flag](https://www.git-town.com//documentation/commands/new-branch-push-flag.md) -
   configures whether new empty branches are pushed to origin
-- [git town main-branch](/documentation/commands/main-branch.md) - displays or
-  sets the main development branch for the current repo
-- [git town offline](/documentation/commands/offline.md) - enables/disables
-  offline mode
-- [git town perennial-branches](/documentation/commands/perennial-branches.md) -
+- [git town main-branch](https://www.git-town.com//documentation/commands/main-branch.md) -
+  displays or sets the main development branch for the current repo
+- [git town offline](https://www.git-town.com//documentation/commands/offline.md) -
+  enables/disables offline mode
+- [git town perennial-branches](https://www.git-town.com//documentation/commands/perennial-branches.md) -
   displays or updates the perennial branches for the current repo
-- [git town pull-branch-strategy](/documentation/commands/pull-branch-strategy.md) -
+- [git town pull-branch-strategy](https://www.git-town.com//documentation/commands/pull-branch-strategy.md) -
   displays or sets the strategy with which perennial branches are updated
-- [git town set-parent-branch](/documentation/commands/set-parent-branch.md) -
-  updates a branch's parent
 
-**Git Town Installation**
+#### Git Town Installation
```

Comment:
Thinking about this section title it seems confusing as its not how to install git town. Not certain what is a better title but think could be useful to discuss something different. Not a hold up for this PR since you're just changing formatting

Reply:
Very good point, especially since there is another file for "installation". I have changed this section to "Git Town setup" for now. The thinking is:

- installation = getting the binary on your computer
- setup = customizing how your computer calls the binary
- configuration = customizing how the binary behaves